data/governments.txt

```diff
@@ -865,6 +865,23 @@ government "Merchant"
 	"friendly hail" "friendly civilian"
 	"hostile hail" "hostile civilian"
 
+government "Merchant (Hijacked)"
+	swizzle 5
+	"attitude toward"
+		"Republic" -.2
+		"Syndicate" -.2
```

Comment:
A value above .05 will allow you to affect permanent change to that faction's rep, while something like -.01 will only make them hostile. Seeing as this is just a government for one-off missions, that rep shouldn't change.

Reply:
Okay, thanks. I did not know that.

Reply:
This is a government used for a single mission; there's no point in having attitude values above +/- .01 since the government is meant to be friendly until attacked.